### PR TITLE
List open and all peer reviews

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: apps/src/code-studio/peer_reviews/PeerReviewDashboard.jsx [hunk 1]
```diff
@@ -1,32 +1,71 @@
 import React from 'react';
 import Spinner from '../pd/workshop_dashboard/components/spinner';
-import EscalatedSubmissions from './EscalatedSubmissions';
+import PeerReviewSubmissions from './PeerReviewSubmissions';
 
 const PeerReviewDashboard = React.createClass({
```
Comment (Hamms) [inline apps/src/code-studio/peer_reviews/PeerReviewDashboard.jsx]: given that createClass is deprecated pending complete removal, now might be a good time to update this to an ES6 class

File: apps/src/code-studio/peer_reviews/PeerReviewDashboard.jsx [hunk 2]
```diff
@@ -1,32 +1,71 @@
...
-    if (this.state.escalatedSubmissions) {
+    if (Object.keys(this.state).length) {
       return (
-        <EscalatedSubmissions
-          submissions={this.state.escalatedSubmissions}
-        />
+        <div>
+          {
+            this.state.escalatedSubmissions && (
+              <PeerReviewSubmissions
```
Comment (Hamms) [inline apps/src/code-studio/peer_reviews/PeerReviewDashboard.jsx]: Should these tables have titles?

File: dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb [hunk 3]
```diff
@@ -1,37 +1,31 @@
...
-
-    escalated_reviews.each do |review|
-      plc_course_unit = review.script.plc_course_unit
+    case params[:filter]
+      when 'escalated'
+        reviews = PeerReview.escalated.limit(50)
+      when 'open'
+        reviews = PeerReview.where(reviewer: nil).limit(50)
+      else
+        reviews = PeerReview.all.limit(50)
```
Comment (Hamms) [inline dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb]: I imagine we're going to want to make this limit controllable by a query parameter at some point; any reason not to go ahead and do that now?

File: dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb [hunk 4]
```diff
@@ -1,37 +1,31 @@
...
-        submitter: review.submitter.name,
-        course_name: plc_course_unit.plc_course.name,
-        unit_name: plc_course_unit.name,
-        level_name: review.level.name,
-        submission_date: review.created_at.strftime("%-m/%-d/%Y"),
-        escalation_date: review.updated_at.strftime("%-m/%-d/%Y"),
-        review_id: review.id
-      }
+    reviews.each do |review|
+      submissions[review.user_level.id] = review.submission_summarize
```
Comment (Hamms) [inline dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb:27]: there are two reviews created for each user_level, right? I can see how that wasn't as much of problem before given the unlikeliness of both reviewers escalating, but it seems like a pretty big issue now.

File: apps/src/code-studio/peer_reviews/PeerReviewDashboard.jsx (hunk 2 above)
Comment (mehalshah) [reply apps/src/code-studio/peer_reviews/PeerReviewDashboard.jsx]: I'll include them

File: dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb (hunk 4 above)
Comment (mehalshah) [reply dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb:27]: Discussed offline

**Pull request merged**